proposals/proposal-template.md

```diff
@@ -0,0 +1,112 @@
+<!---
+Welcome! Thank you for contributing to the U.S. Web Design System.
+Your contributions are vital to our success and we are glad you want to share your ideas.
+
+You can find full instructions for creating a proposal in the uswds-proposals repo:
+https://github.com/amyleadem/uswds-proposals/edit/add-readme-and-template/README.md#submitting-a-new-proposal
+
+More information about contributing to USWDS can be found on the contribution page:
+https://designsystem.digital.gov/about/contribute/
+ -->
+
+## Summary*
+
+_Provide a short paragraph or bulleted list that gives an overview of the proposed component._
+
+## Motivation*
+
+_Explain why we should include this component in USWDS._
+
+- _What user need does this component address?_
+- _What problem does this component solve?_
```

Comment:
I am not sure I understand this comment. 
In case it is relevant - I originally intended these nested questions to be more like prompts or suggestions for filling out the section, rather than mandatory questions to answer. Curious to hear thoughts on that. Do we want these to be mandatory?